common/src/main/cpp/include/extensions/openxr_fb_composition_layer_secure_content_extension_wrapper.h

```diff
@@ -0,0 +1,83 @@
+/**************************************************************************/
+/*  openxr_fb_composition_layer_secure_content_extension_wrapper.h        */
+/**************************************************************************/
+/*                       This file is part of:                            */
+/*                              GODOT XR                                  */
+/*                      https://godotengine.org                           */
+/**************************************************************************/
+/* Copyright (c) 2022-present Godot XR contributors (see CONTRIBUTORS.md) */
+/*                                                                        */
+/* Original contributed implementation:                                   */
+/*   Copyright (c) 2024 Malcolm Nixon                                     */
```

Comment:
Looks like you copied the wrong license header 🙂

Reply:
Fixed in latest push